### ISSUE TITLE
Pipelines to core modules

### ISSUE THREAD

**Issue by plaresmedima** [QIB-Sheffield/WEASEL#41]
Treeview Imports from Pipelines.ViewImage - so Viewimage needs to be in core modules.

**Comment by JSousa-UoL** [comment]
I agree with you, but as mentioned before, scripts like this ViewImage and ViewMetadate were moved from CoreModules to MenuItems at the time (now Pipelines). This happened a couple of months ago as part of a restructuring task.
I'm ok to have a look at moving this back to CoreModules, but I'd like to hear some input from @IMI-TRISTAN. Is this doable with the current structure?

**Comment by IMI-TRISTAN** [comment]
It should be Ok to move these modules to a new folder because the menu
building code takes the name of the module from the XML file and then walks
the Weasel folder structure looking for the location of that module.
